src/main/scala/com/fulcrumgenomics/sv/tools/AggregateSvPileup.scala

```diff
@@ -411,13 +431,29 @@ case class AggregatedBreakpointPileup(id: String,
    * @param targets Optional OverlapDetector for target intervals. If None, target overlap fields are set to false.
    */
   def addTargetOverlap(targets: Option[OverlapDetector[BEDFeature]]): AggregatedBreakpointPileup = {
```

Comment:
This whole function seems kinda whacky to me.  Why even call overlapsAny separately - just get the overlaps, and then ask if you got an empty list back?

Could probably be:

```scala
def addTargetOverlap(targets: Option[OverlapDetector[BEDFeature]]): AggregatedBreakpointPileup = targets match {
    case None     => this
    case Some(od) => 
      val lefts  = od.getOverlaps(left_contig, left_min_pos, left_max_pos)
      val rights = od.getOverlaps(right_contig, right_min_pos, right_max_pos)
      copy(
        left_overlaps_target  = left.nonEmpty,
        right_overlaps_target = rights.nonEmpty,
        left_targets          = lefts.map(_.getName).toSeq.distinct.sorted.mkString(","),
        right_targets         = rights.map(_.getName).toSeq.distinct.sorted.mkString(",")
      )
}
```

Am I missing something?